File: ServiceRequest/Models/ServiceRequest.cs
using ServiceRequest.Models;
using ServiceRequest.Models.Enums;
using System.ComponentModel.DataAnnotations;

namespace ServiceRequest.Models
{
    public class ServiceRequest
    {
            public int Id { get; set; }
           
            public string? RequestNumber { get; set; } 

            public string? Title { get; set; } 
            public string? Description { get; set; } 

            public RequestType RequestType { get; set; }
            public Priority Priority { get; set; }
            public Status Status { get; set; } 

            public string? CreatedBy { get; set; } 
            public string? AssignedTo { get; set; }

            public DateTime CreatedDate { get; set; } 
            public DateTime UpdatedDate { get; set; }
            public ICollection<Attachment>? Attachments { get; set; }



    }
}

File: FrontEnd/service-request/src/app/models/ServiceRequest.ts
import { Priority } from "./Enums/priority.enum";
import { RequestType } from "./Enums/request-type.enum";
import { Status } from "./Enums/status.enum";

export interface ServiceRequest {
 id: number;

  requestNumber?: string;

  title?: string;
  description?: string;

  requestType: RequestType;
  priority: Priority;
  status: Status;

  createdBy?: string;
  assignedTo?: string;

  createdDate: string;   
  updatedDate: string;   
}